tests/integration/SdkTest_test.cpp

```diff
@@ -4559,13 +4561,22 @@ TEST_F(SdkTest, SdkSimpleCommands)
     }
 
     err = synchronousKillSession(0, INVALID_HANDLE);
-    ASSERT_EQ(MegaError::API_ESID, err) << "Kill session for unknown seesions shoud fail with API_ESID (error: " << err << ")";
+    ASSERT_EQ(MegaError::API_ESID, err) << "Kill session for unknown sessions shoud fail with API_ESID (error: " << err << ")";
+
+    // getMiscFlags() -- not logged in
+    logout(0);
+    err = synchronousGetMiscFlags(0);
+    ASSERT_EQ(MegaError::API_OK, err) << "Get misc flags failed (error: " << err << ")";
+
+    // getUserEmail() test
+    ASSERT_NO_FATAL_FAILURE(getAccountsForTest(1));
+
```

Comment:
I think this part is useless here, since the emails has already been tested [here](https://github.com/meganz/sdk/pull/2198/files#diff-eb16573401d197a6ba4d030ed6fa3727R4535) 
```suggestion cpp
```

Reply:
It's needed for the --RESUMESESSIONS case, where we are trying to run the tests more quickly by skipping logout/login inbetween tests.   Because this test cancels it's own session we need to actually log in again.    For the non resume sessions case, we don't care so much about speed.

Reply:
Oh, I see. I didn't realized about the resume sessions mode :+1: